platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -215,67 +214,13 @@ public Schema getSchema(CFMetaData metadata) {
         
         @Override
         public List<IColumn> loadColumns(List<String> names) {
-            boolean[] columnToLoad = getColumnMarker(names);
             IAppendableColumn[] columns = this.actualSchema.createAppendableColumns();
+            boolean[] columnToLoad = getColumnMarker(names);
```

Comment:
In general we prefer `this.` for any local field, including methods. This is actually mandatory in a language like typeScript, and I think it's very useful.

Reply:
Noted! I made some changes